### ISSUE TITLE
cannot pair my Xbox One wireless controller with bluetooth

### ISSUE THREAD

**Issue by timduarte** [fvdhoef/aquarius-plus#36]
I can’t seem to pair my Xbox One wireless controller.
As soon as I turn on the controller by pressing the Round Xbox button ,
the Round Xbox button never stops blinking / flashing.
 It does say it is a wireless XBOX ONE controller.  I have tried other batteries but get the same thing.  
I tried pressing the little pair button on the back, but it does nothing.

**Comment by fvdhoef** [comment]
Only Xbox One S/X controllers support Bluetooth LE out of the box. The older Xbox One controllers only support Bluetooth LE after updating its firmware.

To update your firmware, please follow the guide on:
https://support.xbox.com/en-US/help/hardware-network/controller/update-xbox-wireless-controller

**Comment by timduarte** [comment]
I downloaded the X-BOX Accessories Windows 10 Store app ( free ) and updated the controller. 
So now I press the Round Xbox button ,
the Round Xbox button begins blinking / flashing rapidly.
After about 5 seconds, the controller vibrates twice and then powers off.
I can press the little pair button, but it does not make a difference.
I also tried installing 2 brand new AA batteries, right out of the package. Same thing.
